components/index.rst

```diff
@@ -1,12 +1,19 @@
-PHP Ecommerce Components
-========================
+PHP E-Commerce Components
```

Comment:
IMHO we should name it `PHP Components` and do the same with bundles. We already have Grid component, ResourceBundle, ThemeBundle, MetadataBundle, UserBudle - they don't have anything in common with E-Commerce.